src/librustc/infer/error_reporting/nice_region_error/find_anon_type.rs

```diff
@@ -162,7 +162,7 @@ impl Visitor<'tcx> for FindNestedTypeVisitor<'tcx> {
                             "FindNestedTypeVisitor::visit_ty: LateBound depth = {:?}",
                             debruijn_index
                         );
-                        debug!("self.infcx.tcx.hir().local_def_id(id)={:?}", id);
+                        debug!("self.infcx.tcx.hir().local_def_id_from_node_id(id)={:?}", id);
```

Comment:
Why is is this stuff in the `debug!` string? There are only `DefId`s here anyway AFAICT.

Reply:
I dunno; shall I remove this line?

Reply:
Maybe. Might be useful to look at the history to see who added it or why.

You *could* have `id={:?}, def_id={:?}` instead, using the names in the code specifically.
(that, at least, would be more useful/clear to someone debugging the code in question)